Modules/SaltStackConfig/SaltStackConfig.psm1

```diff
@@ -141,7 +141,9 @@ Function Get-SscData {
   }
 
   try{
-    $output = Invoke-WebRequest -WebSession $global:DefaultSscConnection.SscWebSession -Method POST -Uri "https://$($global:DefaultSscConnection.Name)/rpc" -body $(ConvertTo-Json $body) -ContentType 'application/json'
+    $jsonBody = $(ConvertTo-Json $body -Depth 4 -Compress )

```

Comment:
The default depth limitation is 2.  Due to the arguments passed in via Set-SscMinionKeyState/Remove-SscMinionKeyState a value of at least 3 is required.  I added one extra level for the future, just in case it is needed.